backend/corpora/common/entities/collection.py

```diff
@@ -153,50 +151,15 @@ def to_dict(db_object):
 
         return results
 
-    columns_for_list_collections = {
-        DbCollectionLink: [
-            "link_name",
-            "link_url",
-            "link_type",
-        ],
-        DbCollection: [
-            "id",
-            "name",
-            "visibility",
-            "tombstone",
-            "contact_name",
-            "contact_email",
-            "curator_name",
-            "revised_at",
-            "created_at",
-            "published_at",
-            "description",
-            "publisher_metadata",
-            "revision_of",
-            "tombstone",
-            "owner",  # Needed for determining view permissions
-            "links",
-            "datasets",
-        ],
-        DbDataset: [
-            "id",
-            "curator_tag",
-            "tissue",
-            "assay",
-            "disease",
-            "organism",
-            "tombstone",
-            "processing_status",
-        ],
-        DbDatasetProcessingStatus: ["processing_status"],
-    }
-
     @classmethod
-    def list_collections_for_curation(cls, session: Session, visibility: str = None) -> typing.List[dict]:
+    def list_collections_curation(
+        cls, session: Session, collection_columns: list, visibility: str = None
+    ) -> typing.List[dict]:
         """
         Get a subset of columns, in dict form, for all Collections with the specified visibility. If visibility is None,
         return *all* Collections.
         @param session: the SQLAlchemy session
+        @param collection_columns: the list of columns to be returned (see usage by TransformingBase::to_dict_keep)
```

Comment:
```suggestion
        :param collection_columns: the list of columns to be returned (see usage by TransformingBase::to_dict_keep)
```

Reply:
update the rest of the parameters for consistency.

Reply:
I'm confused,  you want to change this line back to `:`?

Reply:
I thought you had asked to move to `@` recently

Reply:
everything else is using `:`. If we switch to `@` we need to do it all at once to avoid confusion. I don't think the migration to `@` is actually needed.

Reply:
made the change